### ISSUE TITLE
Dispatcher invalidation unclear

### ISSUE THREAD

**Issue by kwin** [AdobeDocs/experience-manager-cloud-manager.en#15]
Issue in help/using/configuring-pipeline.md

From reading the section "Dispatcher Invalidation" it is not clear whether it is flushed/invalidated via ReST request via https://docs.adobe.com/content/help/en/experience-manager-dispatcher/using/configuring/page-invalidate.html#delete-cached-files or it is directly acting on the filesystem. Related to that do the paths refer to filesystem paths or repository paths?
How to remove stuff from the author dispatcher cache?
Which host is being used for the ReST request (if those are used). Is it the "flush" host?

**Comment by justinedelson** [comment]
Hi @kwin.

How invalidation is done should be considered an implementation detail. Is there something you would do with this information? or is this just trivia?

They are repository paths. The file system structure is similarly implementation detail. Clarified with https://github.com/AdobeDocs/experience-manager-cloud-manager.en/pull/16

**Comment by kwin** [comment]
It is crucial to know because in this context `invalidate` vs. `flush` is not explained. Usually `invalidate` only is not possible via https://docs.adobe.com/content/help/en/experience-manager-dispatcher/using/configuring/page-invalidate.html#delete-cached-files/. You will always also flush some content. How does it relate to the statfile level being configured in the dispatcher. This is important info to estimate the impact of the invalidation/flush!

The most important aspect though is which host is being used for flushing (if multiple vhost are being set up with separate dispatcher roots) and especially how to target the author dispatcher cache!

Regarding 
> The file system structure is similarly implementation detail.

this can differ from impl to impl. Only the default is given in https://helpx.adobe.com/experience-manager/kb/ams-dispatcher-manual/understanding-cache.html#cache-dirs but since this can be adjusted it would  have been crucial to know if the paths would have referred to filesystem paths!

**Comment by kwin** [comment]
Also since the HTML/CSS/JS/image/video rendering might change with every software deployment I don't really understand, why flushing both caches fully is not default behaviour for the Cloud Manager!

**Comment by justinedelson** [comment]
> It is crucial to know because in this context invalidate vs. flush is not explained.

Right, these are dispatcher concepts. This might be better raised as an issue on https://github.com/AdobeDocs/experience-manager-dispatcher.en. I don't think it would make sense to duplicate that content here; rather the link is appropriate.

> this can differ from impl to impl.

If the file system structure is changed, the CSE is responsible for ensuring that the actions issued by the AMS Command & Control system (of which Cloud Manager is a client) continues to be able to invalidate/flush correctly.

> Also since the HTML/CSS/JS/image/video rendering might change with every software deployment I don't really understand, why flushing both caches fully is not default behaviour for the Cloud Manager!

If you would like to make a feature request, please work with Adobe support. This is not the right forum.

**Comment by kwin** [comment]
Ok, let's just boil it down to the questions:

- Which paths have to be given (file path, repo path (mapped or unmapped))?
- How to target author dispatcher cache?
- Add links to pages which explain `flush` vs `invalidate`

**Comment by bohnertchris** [comment]
CQDOC-16093
